tests/test_meshtastic_utils_connect_paths.py

```
@@ -39,6 +40,63 @@ def test_connect_meshtastic_returns_existing_client(reset_meshtastic_globals):
+    worker_errors: list[BaseException] = []
+    worker_results: list[Any | None] = [None, None]
+
+    def _tcp_constructor(*_args: Any, **_kwargs: Any) -> Any:
+        constructor_entered.set()
+        assert allow_constructor_return.wait(timeout=1.0)
+        return created_client
+
+    def _run_connect(index: int) -> None:
+        try:
+            worker_results[index] = connect_meshtastic(passed_config=config)
+        except BaseException as exc:  # pragma: no cover - exercised only on failure
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `BaseException`

(BLE001)